src/hooks/useCombobox/index.js

```diff
@@ -49,10 +50,12 @@ function useCombobox(userProps = {}) {
   const initialState = getInitialState(props)
 
   // Reducer init.
-  const [
-    {isOpen, highlightedIndex, selectedItem, inputValue},
-    dispatch,
-  ] = useControlledReducer(downshiftUseComboboxReducer, initialState, props)
+  const [state, dispatch] = useControlledReducer(
+    downshiftUseComboboxReducer,
+    initialState,
+    props,
+  )
+  const {isOpen, highlightedIndex, selectedItem, inputValue} = state
```

Comment:
Mostly formatting changes. I just wanted to be able to create the `state` variable so I could use it in `useLatestRef`.